Adding loot cred with empty username and password crashes client

**Describe the bug**

When using the `loot creds` command to add a USER_PASSWORD credential, if the user and password is empty, then the client will crash on subsequent commands that query the loots database.

**To Reproduce**

Steps to reproduce the behavior:
1. Start sliver client
2. Run 'loot creds -n somename'
3. Select USER_PASSWORD
4. Press enter for an empty username
5. Press enter for an empty password
6. Receive success message for loot added
7. Type in one of the following commands: 'loot creds', 'loot rm', or 'loot' (any command that lists loot in server's db)

```
sliver > loot creds -n credname

? Choose a credential type: USER_PASSWORD
? Username:
? Password:

[*] Successfully added loot to server (2aa0c04c-715e-4204-a97a-f38f92239194)
```

**Expected behavior**

Client should handle empty credentials gracefully or empty credentials should be rejected when added.

**Screenshots**

```
sliver > loot

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0xc56769]

goroutine 1 [running]:
github.com/bishopfox/sliver/client/command/loot.PrintAllLootTable({0x1988780, 0xc000758ea0}, 0xc0007f2780)
	github.com/bishopfox/sliver/client/command/loot/loot.go:170 +0x3a9
```

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS] Debian Linux
 - Version [e.g. v0.0.5] 5.10.127-1

**Additional context**

In my environment, there were no other loot entries in the database. The server did not crash so I believe this issue is limited to the client.

--- a/client/command/loot/add-credential.go
+++ b/client/command/loot/add-credential.go
@@ -55,10 +55,20 @@ func LootAddCredentialCmd(ctx *grumble.Context, con *console.SliverConsoleClient
 	switch credType {
 	case clientpb.CredentialType_USER_PASSWORD.String():
 		loot.CredentialType = clientpb.CredentialType_USER_PASSWORD
-		usernamePrompt := &survey.Input{Message: "Username: "}
-		survey.AskOne(usernamePrompt, &loot.Credential.User)
-		passwordPrompt := &survey.Input{Message: "Password: "}
-		survey.AskOne(passwordPrompt, &loot.Credential.Password)
+		for loot.Credential.User == "" {
+			usernamePrompt := &survey.Input{Message: "Username: "}
+			survey.AskOne(usernamePrompt, &loot.Credential.User)
+			if loot.Credential.User == "" {
+				con.Println("Username is required")
+			}
+		}
+		for loot.Credential.Password == "" {
+			passwordPrompt := &survey.Input{Message: "Password: "}
+			survey.AskOne(passwordPrompt, &loot.Credential.Password)
+			if loot.Credential.Password == "" {
+				con.Println("Password is required")
+			}
+		}
 	case clientpb.CredentialType_API_KEY.String():
 		loot.CredentialType = clientpb.CredentialType_API_KEY
 		usernamePrompt := &survey.Input{Message: "API Key: "}

--- a/client/command/loot/helpers.go
+++ b/client/command/loot/helpers.go
@@ -176,10 +176,14 @@ func SelectLoot(ctx *grumble.Context, rpc rpcpb.SliverRPCClient) (*clientpb.Loot
 	buf := bytes.NewBufferString("")
 	table := tabwriter.NewWriter(buf, 0, 2, 2, ' ', 0)
 	for _, loot := range allLoot.Loot {
-		if loot.Name == loot.File.Name {
+		filename := ""
+		if loot.File != nil {
+			filename = loot.File.Name
+		}
+		if loot.Name == filename {
 			fmt.Fprintf(table, "%s\t%s\t%s\t\n", loot.Name, loot.Type, loot.LootID)
 		} else {
-			fmt.Fprintf(table, "%s\t%s\t%s\t%s\t\n", loot.Name, loot.File.Name, loot.Type, loot.LootID)
+			fmt.Fprintf(table, "%s\t%s\t%s\t%s\t\n", loot.Name, filename, loot.Type, loot.LootID)
 		}
 	}
 	table.Flush()

--- a/client/command/loot/loot.go
+++ b/client/command/loot/loot.go
@@ -166,8 +166,13 @@ func PrintAllLootTable(stdout io.Writer, allLoot *clientpb.AllLoot) {
 		strings.Repeat("=", len("File Name")),
 		strings.Repeat("=", len("UUID")),
 	)
+
 	for _, loot := range allLoot.Loot {
-		fmt.Fprintf(table, "%s\t%s\t%s\t%s\t\n", lootTypeToStr(loot.Type), loot.Name, loot.File.Name, loot.LootID)
+		filename := ""
+		if loot.File != nil {
+			filename = loot.File.Name
+		}
+		fmt.Fprintf(table, "%s\t%s\t%s\t%s\t\n", lootTypeToStr(loot.Type), loot.Name, filename, loot.LootID)
 	}
 
 	table.Flush()

